roles/disaster_recovery/examples/dr_play.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Setup oVirt environment
+  hosts: localhost
+  connection: local
+  roles:
+     - disaster_recovery
```

Comment:
missing collection usage( `collections: ovirt.ovirt` )

Reply:
Done.
Commit "updating the role name" was updated with the fix.